docs/concepts/scheduled-jobs.md

```diff
@@ -0,0 +1,37 @@
+(scheduled-jobs)=
+# Scheduled Jobs
+
+Often times you may want to repeadedly run a `Job` on a regular schedule. For example, fetching from a datasource every morning, compiling an analytics report every month, or detecting model drift every hour.
```

Comment:
Oftentimes you may want to repeatedly run a `job` on a regular schedule. For example, fetching from a datasource every morning, compiling an analytics report every month, or detecting model drift every hour.